apps/uci/web/src/components/common/starred-chat-list/index.tsx

```diff
@@ -118,31 +118,28 @@ export const StarredChatList: FC<{ user: User }> = ({ user }) => {
                     <FileCard
                         // @ts-ignore
                         file={{
-                            name: 'sample',
-                            size: 12345
+                            name: config.Starred_Chat.file_name,
+                            size:config.Starred_Chat.size
```

Comment:
I'm not sure if here any configuration is required . Use config for important stuff only